libs/rand/sampling.go

```diff
@@ -24,9 +24,16 @@ type Candidate interface {
 // Returns:
 // samples - A randomly selected candidate from a set of candidates. NOTE that the same candidate may have been
 // selected in duplicate.
+// TODO: Change to a function that returns an error, as it is not desirable for the function to end with a panic.
```

Comment:
Why do you leave TODO?

Reply:
Currently, panic is returned, and I thought that returning error as a libs function would be more versatile.
I'll fix it if it is wrong as how to leave TODO

Reply:
Then, it would be better to fix it instead of leaving `TODO`. I agree with your thoughts. How about @torao?

Reply:
IMO, `TODO` can stay forever.

Reply:
It's certainly not something that should be left in the code.
I'll remove it and leave it in the other document.

Reply:
Please make an issue. Thanks